orcestra_book/plans/HALO-20240827a.md

```diff
@@ -144,8 +145,7 @@ c_mid = c_south.towards(c_north).assign(label = "c_mid")
 x_wp1  = c_mid.towards(c_north,fraction=2/5.).assign(label = "x_wp1")
 
 # ec underpass
-ec_under = c_north.towards(ec_north,fraction=1./2.).assign(label = "ec_under")
-ec_under.assign(time=str(ec_time_at_lat(ec_track, ec_under.lat).values)+"Z")
+ec_under = c_north.towards(ec_north,fraction=1./3.).assign(label = "ec_under")
```

Comment:
This places ec_under on the great circle between the two points. An easy but slightly hacky way of solving this is to use the find_ec_lon function again:

ec_under_gc = c_north.towards(ec_north,fraction=1./3.)
ec_under = LatLon(ec_under_gc.lat, find_ec_lon(ec_under_gc.lat, ec_lons, ec_lats), label = "ec_under")